android/src/main/java/com/bitmovin/player/reactnative/SourceModule.kt

```diff
@@ -53,6 +53,7 @@ class SourceModule(private val context: ReactApplicationContext) : ReactContextB
         nativeId: NativeId,
         drmNativeId: NativeId?,
         config: ReadableMap?,
+        sourceRemotePlaybackConfig: ReadableMap?,
```

Comment:
Let's add it to the function documentation and mention that this is unused/unsupported on Android. Holds for both functions.

Reply:
Fixed as part of the renaming in 852d26f

Reply:
Can you please also mention that this is unused/unsupported on Android? 😇

Reply:
missed that part, fixed in 2fe285f